python/ray/actor.py

```diff
@@ -185,7 +185,7 @@ class ActorClass(object):
             task.
         _resources: The default resources required by the actor creation task.
         _actor_method_cpus: The number of CPUs required by actor method tasks.
-        _last_job_id_exported_for: The ID of the job of the last Ray
+        _last_session_index: The index of the last exported session.
```

Comment:
I'd rename `worker._session_index` to `_current_session`, and change it to a pair of `session_index` and `job_id`. 
Then here, we can define a `_last_export_session` variable, and compare it with `worker._current_session`.